pkg/asset/internal/templates.go

```diff
@@ -145,6 +145,7 @@ spec:
         component: kube-apiserver
       annotations:
         checkpointer.alpha.coreos.com/checkpoint: "true"
+        scheduler.alpha.kubernetes.io/critical-pod: ''
```

Comment:
Ah right. The annotation is basically so it can check if it isn't being scheduled (and would evict other work).

Yeah, if the daemon controller respects it (and maybe later kubelet enforces taints) might not be bad to have the toleration already set.